recipes/r-rnexml/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = '2.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rnexml
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: RNeXML_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/RNeXML_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/RNeXML/RNeXML_{{ version }}.tar.gz
+  sha256: 97f472067e42ad17a504f3ff9180f2575e47507a794735a23e7a69a54b3ddcc2
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-xml >=3.95
+    - r-ape >=3.1
+    - r-dplyr >=0.5.0
+    - r-httr >=0.3
+    - r-lazyeval >=0.1.0
+    - r-plyr >=1.8
+    - r-reshape2 >=1.2.2
+    - r-stringr >=1.0
+    - r-tidyr >=0.3.1
+    - r-uuid >=0.1_1
+    - r-xml2
+
+  run:
+    - r-base
+    - r-xml >=3.95
+    - r-ape >=3.1
+    - r-dplyr >=0.5.0
+    - r-httr >=0.3
+    - r-lazyeval >=0.1.0
+    - r-plyr >=1.8
+    - r-reshape2 >=1.2.2
+    - r-stringr >=1.0
+    - r-tidyr >=0.3.1
+    - r-uuid >=0.1_1
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('RNeXML')"  # [not win]
+    - "\"%R%\" -e \"library('RNeXML')\""  # [win]
```

Comment:
only need 

```
- $R -e "library('RNeXML')"
```

Reply:
Thanks for the clarification! I didn't realize that all windows lines should be removed. I've updated the recipe as you suggested.